sdk/storage/Azure.Storage.DataMovement/tests/PauseResumeTransferTests.cs

```diff
@@ -53,47 +122,72 @@ public class PauseResumeTransferTests : DataMovementBlobTestBase
         /// By default in this function an event arguement will be added to the options event handler
         /// to detect when the upload has finished.
         /// </summary>
-        private async Task<DataTransfer> CreateLongTransferAsync(
+        private async Task<DataTransfer> CreateSingleLongTransferAsync(
             TransferManager manager,
-            string sourceDirectory,
-            BlobContainerClient destinationContainer,
+            TransferType transferType = TransferType.Upload,
+            string localDirectory = default,
+            BlobContainerClient sourceContainer = default,
+            BlobContainerClient destinationContainer = default,
             StorageResource sourceResource = default,
             StorageResource destinationResource = default,
             SingleTransferOptions singleTransferOptions = default,
             long size = Constants.MB)
         {
-            sourceResource ??= await CreateLocalFileSourceResourceAsync(size, sourceDirectory);
-            destinationResource ??= CreateBlobDestinationResourceAsync(destinationContainer);
+            Argument.AssertNotNull(manager, nameof(manager));
+            if (sourceResource == default && destinationResource == default)
+            {
+                (StorageResource source, StorageResource dest) = await CreateStorageResourcesAsync(
+                    transferType: transferType,
+                    size: size,
+                    localDirectory: localDirectory,
+                    sourceContainer: sourceContainer,
+                    destinationContainer: destinationContainer);
+                sourceResource = source;
+                destinationResource = dest;
+            }
+            else if ((sourceResource == default && destinationResource != default) ||
+                    (sourceResource != default && destinationResource == default))
+            {
+                throw new ArgumentException($"Both {nameof(sourceResource)} or {nameof(destinationResource)} must be specified, " +
+                    $"if only one is specified.");
+            }
 
             // Act
             return await manager.StartTransferAsync(sourceResource, destinationResource, singleTransferOptions);
         }
 
-        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/35439")]
+        [LiveOnly]
```

Comment:
nit: For each one of these "large" tests. Can we add a comment next to this line saying why its LiveOnly (too big to record).

Also, remove commented ignores?

Reply:
Ah I left them in just in case the live test were going to be unkind, and looks like they are. Let me up the size and see if it helps the flakiness first.

Reply:
Hmm yeah, it might be tough to get the pipeline to take "long enough". Especially with a single transfer. The pipeline runs in Azure so can do insanely fast network operations. Probably takes milliseconds to 100 MiB. We may need to find a way to mock stuff or inject an artificial delay somehow.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/35558